pkg/apis/toolchain/v1alpha1/toolchainstatus_types.go

```diff
@@ -0,0 +1,201 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// These are valid status condition reasons for Toolchain status
+const (
+	// overall status condition reasons
+	ToolchainStatusReasonAllComponentsReady = "AllComponentsReady"
```

Comment:
We try to follow the following format of the reasons:
`<Kind><ReasonString>Reason = "<ReasonString>"`
For example:
`UserAccountUnableToCreateIdentityReason = "UnableToCreateIdentity"`
For this particular reason it should be:
```suggestion
	ToolchainStatusAllComponentsReadyReason = "AllComponentsReady"
```

Lets follow this rule for the sake of constancy.

Reply:
Ah, ok makes sense. I've updated them.